components/clearbutton/index.css

```diff
@@ -46,6 +46,7 @@
 		--mod-clear-button-background-color-key-focus: transparent;
 	}
 
+	&.spectrum-ClearButton--staticWhite,
```

Comment:
```suggestion
	/* @deprecated .spectrum-ClearButton--overBackground */
	&.spectrum-ClearButton--staticWhite,
```

Reply:
Sorry missed that overBackground was being deprecated

Reply:
No problem! I'll get that incorporated. ✨